enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationService.java

```diff
@@ -400,12 +402,27 @@ private void handleDocumentStatus(
                 .findAny()
                 .ifPresent(docVerification -> {
                     logger.debug("At least one document is {}, ID: {}, {}", status, docVerification.getId(), ownerId);
-                    idVerification.setErrorDetail(docVerification.getRejectReason());
+                    idVerification.setErrorDetail(fetchErrorDetailShort(docVerification));
```

Comment:
This change is incorrect.

Please see:
https://github.com/wultra/enrollment-server/blob/develop/enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/entity/IdentityVerificationEntity.java#L50

We want to continue using errors like this in `IdentityVerificationEntity`, because the value may be consumed by DWH for reporting.

I realize there are many other cases when this is done incorrectly, but let's not spread this error. I will raise a ticket for the other cases.